fuzz/src/main.rs

```diff
@@ -0,0 +1,103 @@
+#![no_main]
+
+use arbitrary::Arbitrary;
+use evm::{
+	backend::{MemoryAccount, MemoryBackend, MemoryVicinity},
+	executor::stack::{MemoryStackState, StackExecutor, StackSubstateMetadata},
+	Config,
+};
+use primitive_types::{H160, U256};
+use std::{collections::BTreeMap, str::FromStr};
+
+const NB_ACCOUNTS: u8 = 5;
+const BALANCES: U256 = U256::MAX;
+
+// Fuzzer input
+#[derive(Arbitrary, Debug)]
+struct EvmInput<'a> {
+	transact_from: u8,
+	transact_to: u8,
+	transact_value: u64,
+	smart_contract_code: &'a [u8],
+	call_input: &'a [u8],
+}
+
+// Fuzzing harness macro, will create the targets for honggfuzz, libfuzzer and afl++
+ziggy::fuzz!(|evminput: EvmInput| {
+	// If we're not fuzzing, we print out information about the input
+	#[cfg(not(fuzzing))]
+	{
+		println!("Running input:");
+		println!("{evminput:?}");
+	}
+
+	let config = Config::frontier();
+
+	let vicinity = MemoryVicinity {
+		gas_price: U256::one(), // Should this be zero instead?
+		origin: H160::default(),
+		block_hashes: Vec::new(),
+		block_number: Default::default(),
+		block_coinbase: Default::default(),
+		block_timestamp: Default::default(),
+		block_difficulty: Default::default(),
+		block_gas_limit: Default::default(),
+		chain_id: U256::one(),
+		block_base_fee_per_gas: U256::one(), // Should this be zero instead?
+	};
+
+	// Create initial state with NB_ACCOUNTS accounts
+	let mut state = BTreeMap::new();
+	for i in 0..NB_ACCOUNTS {
+		state.insert(
+			H160::from_str(&format!("0x000000000000000000000000000000000000000{}", i)).unwrap(),
```

Comment:
I've added a commit to address this issue. This has to be changed everywhere, such that the fuzzer will always use existing addresses. The fix just uses the MSB instead of the LSB for i

Reply:
I don't think anything more will be found by the fuzzer with `>= 10` accounts. I changed it again to avoid this address. That said, letting the fuzzer do a transaction within this range could be interesting if we want to fuzz precompiles too. In this fuzzer I did not include precompiles (see l.821) yet.

Let me know if that solves the issue